src/components/ClaimForm.js

```diff
@@ -151,6 +151,7 @@ class ClaimForm extends Component {
     if (!this.props.isClaimCodeValid) return false;
     if (!!this.state.claim.codeError) return false;
     if (!this.state.claim.healthFacility) return false;
+    if (this.state.claim.visitType==="R" && !this.state.claim.referHF) return false;
```

Comment:
Similar suggestion - please create a seperate constant with "R" string and reuse it here. Moreover, use whitespaces as suggested before.